addons/hr_work_entry_holidays/tests/test_performance.py

```diff
@@ -32,7 +32,7 @@ def test_performance_leave_validate(self):
         self.richard_emp.generate_work_entries(date(2018, 1, 1), date(2018, 1, 2))
         leave = self.create_leave(datetime(2018, 1, 1, 7, 0), datetime(2018, 1, 1, 18, 0))
 
-        with self.assertQueryCount(__system__=114, admin=115):  # com 96/97
+        with self.assertQueryCount(__system__=117, admin=118):  # com 96/97
```

Comment:
Mmmh ? Why this change ?

Reply:
It is also a consequence to adding the digest module as dependency to hr :slightly_smiling_face: 
Similarly to the other test I updated, a specific module is now loaded before `hr_work_entry_holidays` because of the digest dependency in `hr` and the execution flow has slightly changed because of that.
I've been investigating for a few hours to know exactly what's happening but I can't seem to grasp which module is adding the queries. :confused: I always get a query count lower than 117 :upside_down_face:

Reply:
Arf, actually those tests should be made post_install, otherwise they depend on db state. So yes just update with an explanation that module loading changed, and that ideally those tests should become post_install in order to be more deterministic :)